convert2rhel/breadcrumbs.py

```diff
@@ -81,11 +83,19 @@ def collect_early_data(self):
         self._set_source_os()
         self._set_started()
 
-    def finish_collection(self, success=False):
-        """Set the final data for breadcrumbs after the conversion ends."""
+    def finish_collection(self, success=False, action="convert"):
```

Comment:
We could probably set action automatically from the command line flags/environment variable instead of having to pass it in but I think we could do that in a later PR (so we get this working for now.)